brb

```diff
@@ -47,7 +65,17 @@ def main(fname, keys):
     except:
         logging.warn('Unable to fetch wellname')
         name = fname
+
+    conf = None
+    try:
+        with open('config.yml', 'r') as _f:
+            conf = yaml.load(_f)
+    except:
+        logging.warn('Could not read config file')
```

Comment:
her vil du kanskje (for egen del) ha `except Exception as err: ...warn('... %s' % err)`.

Reply:
mhm, denne biter vel over minst to ting (både om filen ikke eksisterer og hvis den ikke er konfigurert rett)

Reply:
Ja, det er flere grunner til at en exception kan kastes her, derfor er det spesielt viktig at du skriver ut årsaken.